From 5d8f33cf982c3a391460e97397164931120cca86 Mon Sep 17 00:00:00 2001
From: Marcel Huber <marcel.huber@hsr.ch>
Date: Thu, 9 Jan 2014 08:57:02 +0100
Subject: inhibit bzr errors during build


diff --git a/Makefile b/Makefile
index cb1ca18..ff35793 100644
--- a/Makefile
+++ b/Makefile
@@ -15,7 +15,7 @@ INSTALL_EXE=${INSTALL} -D
 INSTALL_DATA=${INSTALL} -m 0644 -D
 
 build: etckeeper.spec
-	-./etckeeper-bzr/__init__.py build || echo "** bzr support not built"
+	-./etckeeper-bzr/__init__.py build 2>/dev/null || echo "** bzr support not built"
 	
 install:
 	mkdir -p $(DESTDIR)$(etcdir)/etckeeper/ $(DESTDIR)$(vardir)/cache/etckeeper/
@@ -39,7 +39,7 @@ endif
 ifeq ($(HIGHLEVEL_PACKAGE_MANAGER),zypper)
 	$(INSTALL_DATA) zypper-etckeeper.py $(DESTDIR)$(prefix)/lib/zypp/plugins/commit/zypper-etckeeper.py
 endif
-	-./etckeeper-bzr/__init__.py install --root=$(DESTDIR) ${PYTHON_INSTALL_OPTS} || echo "** bzr support not installed"
+	-./etckeeper-bzr/__init__.py install --root=$(DESTDIR) ${PYTHON_INSTALL_OPTS} 2>/dev/null || echo "** bzr support not installed"
 	echo "** installation successful"
 
 clean: etckeeper.spec
-- 
1.8.5.2

